### PR TITLE
[LLM] Quickstart and how-tos

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - Select the desired model for your deployment from the available options:
+        - Llama-2-70b
+        - Mixtral-8x7B-Instruct-v0.1
+        - WizardLM-70b-V1.0
+        <Message type="note">
+          Some models may require acceptance of an end-user license agreement. If prompted, review the terms and conditions and accept the license accordingly.
+        </Message>
+    - Choose the geographical **region** for the deployment.
+    - Specify the GPU Instance type to be used with your deployment.
+4. Enter a **name** for the deployment, along with an optional description and tags to aid in organization.
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/create-deployment.mdx:26]:
```suggestion
4. Enter a **name** for the deployment and, optionally, description and tags.
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+5. Configure the **network** settings for the deployment:
+    - Enable **Private Network** for secure communication and restricted availability within Private Networks. Choose an existing Private Network from the drop-down list, if applicable.
+    - Enable **Public Network** to access resources via the public IP address of the GPU Instance.
+    <Message type="important">
+      - Once configured, network settings cannot be modified after deployment creation.
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/create-deployment.mdx:31]:
```suggestion
      -  It is not possible to change network settings after the deployment creation. 
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+dates:
+  validation: 2024-03-06
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Once you have finished your inference tasks you can delete your deployment from the Scaleway console.
+
+<Macro id="iam-requirements" />
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:18]: This will be replaced with <Macro id="requirements" />

Comment (jcirinosclwy) [reply ai-data/llm-inference/how-to/delete-deployment.mdx:18]: Macro id="requirements"

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:21]:
```suggestion
  - A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  - You have a [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:22]:
```suggestion
  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Choose a deployment either by clicking its name or selecting **More info** from the drop-down menu represented by the icon <Icon name="more" /> to access the deployment dashboard.
+3. Click the **Settings** tab of your deployment to display additional settings.
+4. Click **Delete deployment** to delete your deployment.
+5. Type **DELETE** to confirm and click **Delete deployment** to delete your deployment. 
+ 
+<Message type="important">
+  Deleting a deployment will erase all associated data. Exercise caution before proceeding with this action.
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:33]:
```suggestion
  Deleting a deployment is a permanent action and erases all its associated data.
```

File: ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+categories:
+  - ai-data
+---
+
+Allowed IPs define remote access restrictions for your Database Instance. The default entry `0.0.0.0/0` enables any host to establish a connection.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx:21]:
```suggestion
  - A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  - You have a [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx:22]:
```suggestion
  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+dates:
+  validation: 2024-03-06
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Monitoring allows you to monitor your deployment.
+
+<Macro id="iam-requirements" />
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:21]:
```suggestion
  - A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Monitoring allows you to monitor your deployment.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:20]:
```suggestion
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+  - You have [enabled your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:23]:
```suggestion
  - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+</Message>
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:24]:
```suggestion
```

File: ai-data/llm-inference/how-to/monitor-deployment.mdx [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+
+## How to monitor your LLM dashboard
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dasdhboard.
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/monitor-deployment.mdx:30]:
```suggestion
2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dashboard.
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Once you have finished your inference tasks you can delete your deployment from the Scaleway console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:20]:
```suggestion
```

File: ai-data/llm-inference/how-to/delete-deployment.mdx [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+</Message>
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/delete-deployment.mdx:23]:
```suggestion
```

File: ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+  posted: 2024-03-06
+categories:
+  - ai-data
+---
+
+Allowed IPs define remote access restrictions for your Database Instance. The default entry `0.0.0.0/0` enables any host to establish a connection.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/how-to/manage-allowed-ip-addresses.mdx:20]:
```suggestion
```

File: ai-data/llm-inference/quickstart.mdx [hunk 18]
```diff
@@ -10,3 +10,67 @@ categories:
...
+## How to delete a deployment
+
+1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
+2. Choose a deployment either by clicking its name or selecting **More info** from the drop-down menu represented by the icon <Icon name="more" /> to access the deployment dashboard.
+3. Click the **Settings** tab of your deployment to display additional settings.
+4. Click **Delete deployment** to delete your deployment.
+5. Type **DELETE** to confirm and click **Delete deployment** to delete your deployment. 
+ 
+<Message type="important">
+  Deleting a deployment will erase all associated data. Exercise caution before proceeding with this action.
```
Comment (jcirinosclwy) [inline ai-data/llm-inference/quickstart.mdx:75]:
```suggestion
  Deleting a deployment is a permanent action and erases all its associated data.
```

**Review by jcirinosclwy — APPROVED** [review]

File: ai-data/llm-inference/how-to/delete-deployment.mdx (hunk 3 above)
Comment (bene2k1) [reply ai-data/llm-inference/how-to/delete-deployment.mdx:18]:
```suggestion
<Macro id="requirements" />
```

**Pull request merged**